### PR TITLE
Mclag domain additional configuration. - b - #63

### PR DESCRIPTION
**Pull request by vivek-metricdust**
related [#63](https://github.com/STORDIS/orca_nw_lib/issues/63)

added functionality to take configuration like 
- session_vrf
- keepalive_int
- session_timeout
- delay_restore

added functionality where only domain id and any other felid can be passed to trigger mclag addition
added default values
added test case to check mclag creation only with domain id and one other param

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -13,7 +13,7 @@ djangorestframework = "^3.14.0"
 django-cors-headers = "^3.14.0"
 pytest = "^7.3.2"
 pytest-django = "^4.7.0"
-orca-nw-lib = "^1.3.17"
+orca-nw-lib = "^1.3.18"
```
Comment (kamalkrbh) [inline pyproject.toml:16]: @vivek-metricdust, Should be +1

File: network/mclag.py [hunk 2]
```diff
@@ -129,11 +133,15 @@ def device_mclag_list(request):
...
-                        session_vrf=session_vrf
+                        session_vrf=session_vrf,
+                        keepalive_int = keepalive_interval,
+                        session_timeout = session_timeout,
+                        delay_restore = delay_restore,
                     )
                     add_msg_to_list(result, get_success_msg(request))
                 except Exception as err:
                     add_msg_to_list(result, get_failure_msg(err, request))
+                    print(traceback.format_exc())
```
Comment (kamalkrbh) [inline network/mclag.py:144]: @vivek-metricdust , Avoid prints.

File: network/test/test_mclag.py [hunk 3]
```diff
@@ -42,6 +42,24 @@ def test_mclag_config(self):
...
+        }
+        
+        response = self.put_req("device_mclag_list", request_body)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        response = self.get_req(
+            "device_mclag_list", {"mgt_ip": device_ip_1, "domain_id": self.domain_id}
+        )
+
+        self.assertEqual(response.json().get("domain_id"), self.domain_id)
```
Comment (kamalkrbh) [inline network/test/test_mclag.py:61]: @vivek-metricdust , assert session timeout as well.

File: network/test/test_mclag.py [hunk 4]
```diff
@@ -42,6 +42,24 @@ def test_mclag_config(self):
...
+        self.remove_mclag(device_ip_1)
```
Comment (kamalkrbh) [inline network/test/test_mclag.py:62]: @vivek-metricdust, Test case doesn't test other parametrers like fast_convergence ,session_vrf ,keepalive_interval ,session_timeout ,delay_restore etc. All parameters should be coverred by tests, also whereever possible not only test them when creating mclag also write test for updating them afterwards.

File: network/mclag.py (hunk 2 above)
Comment (vivek-metricdust) [reply network/mclag.py:144]: removed

File: pyproject.toml (hunk 1 above)
Comment (vivek-metricdust) [reply pyproject.toml]: Updated to new version 1.3.19

File: network/test/test_mclag.py (hunk 3 above)
Comment (vivek-metricdust) [reply network/test/test_mclag.py:157]: assertion added for timeout as well.

File: network/test/test_mclag.py (hunk 4 above)
Comment (vivek-metricdust) [reply network/test/test_mclag.py:161]: new test cases added in [dee43c8](https://github.com/STORDIS/orca_backend/pull/47/commits/dee43c833f9f50aa4f3f944b0e47df9442228ba4)

**Comment by vivek-metricdust** [comment]
All the comments has been resolved, PR is up to date and test cases are also passing

**Pull request merged**